data/categories.txt

```diff
@@ -785,7 +785,7 @@ shop-car
 en:car shop|U+1F697|U+1F698|U+1F699
 ru:автосалон|магазин
 uk:автосалон|магазин
-de:Auto-Shop|Laden
+de:Autohaus
```

Comment:
Here is a fresh documentation: https://github.com/mapsme/omim/wiki/categories.txt <https://github.com/mapsme/omim/wiki/categories.txt>

> On Feb 1, 2016, at 18:56, malenki <notifications@github.com> wrote:
> 
> In data/categories.txt <https://github.com/mapsme/omim/pull/1735#discussion_r51435599>:
> 
> > @@ -785,7 +785,7 @@ shop-car
> >  en:car shop|U+1F697|U+1F698|U+1F699
> >  ru:автосалон|магазин
> >  uk:автосалон|магазин
> > -de:Auto-Shop|Laden
> > +de:Autohaus
> Ok. There should be some documentation somewhere about how this file is processed. My lack of knowledge also resulted in removing some of the "Sehenswürdigkeit" below.
> 
> Hint: "Geschäft" would IMHO be a better category name then "Laden"
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/1735/files#r51435599>.
>